tmol/tests/types/test_torch.py

```diff
@@ -121,7 +121,7 @@ def test_array_validation(example):
             assert not spec.validate(v)
 
 
-invalid_dtypes = [numpy.dtype([("coord", float, 3), ("val", int)]), numpy.complex, "c8"]
+invalid_dtypes = [numpy.dtype([("coord", float, 3), ("val", int)]), complex, "c8"]
```

Comment:
I'm trying to puzzle out: What is triggering the need for this change? I don't see any updates to the env.yml file that might cause a numpy version change.

Reply:
Not needed on test server, but I did some runs with newer versions of numpy and it triggered an error.